### ISSUE TITLE
Add a rawEnv reporter

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1534]
This reporter would return the raw results and the env variables in a single object. Something like this:

```js
axe.addReporter('rawEnv', function(raw, options, callback) {
	if (typeof options === 'function') {
		callback = options;
		options = {};
	}
	const env = helpers.getEnvironmentData();
	callback({ raw, env });
});

```

**Comment by padmavemulapati** [comment]
QA Reviewed.
